app/views/moderations/actions_panel.html.erb

```diff
@@ -154,22 +154,55 @@
       </button>
       <% end %>
       <a href="<%= URL.url %>/community-moderation#how-to-rate-the-experience-level-of-a-post" target="_blank" rel="noopener">
-        <span class="how-this-works-section">
+        <span class="additional-subtext-section py-3">
           <%= t("views.moderations.actions.how") %>
         </span>
       </a>
     </div>
 
     <% if current_user.any_admin? && @moderatable.published %>
-      <div class="unpublish-article">
-        <button
-          class="crayons-btn crayons-btn--danger"
-          id="unpublish-article-btn"
-          data-article-id="<%= @moderatable.id %>"
-          data-article-author="<%= @moderatable.username %>"
-          data-article-slug="<%= @moderatable.slug %>">
-          <%= t("views.moderations.actions.unpublish") %>
-        </button>
+      <button aria-haspopup="true" aria-expanded="false" aria-controls="admin-action-options" class="other-things-btn admin-action" type="button" data-other-things-type="admin-action" aria-label="<%= t("views.moderations.actions.admin.aria_label") %>">
+        <div class="label-wrapper">
+          <div class="icon circle centered-icon">
+            <%= inline_svg_tag("admin.svg", aria: true, title: t("views.moderations.actions.experience.icon")) %>
```

Comment:
```suggestion
            <%= inline_svg_tag("admin.svg", aria_hidden: true, title: t("views.moderations.actions.admin.icon")) %>
```
Since the icon isn't needed to provide the button description, it's better UX for screen reader users to hide it here

Also - I think the title was pointing at the wrong i18n key? It was getting "Book" instead of the "Crown" text added in the yml